pandas/tests/extension/base/methods.py

```diff
@@ -704,3 +704,13 @@ def test_equals(self, data, as_series, box):
     def test_equals_same_data_different_object(self, data):
         # https://github.com/pandas-dev/pandas/issues/34660
         assert pd.Series(data).equals(pd.Series(data))
+
+    def test_round(self, data):
+        if not data.dtype._is_numeric:
+            pytest.skip("Round is only valid for numeric dtypes")
+        result = pd.Series(data).round()
+        expected = pd.Series(
```

Comment:
any reason for np.round vs round?  id expect the builtin to work in more cases.  in fact could just make this `expected` a non-performant base class implementation

Reply:
forgot that existed! nice idea.